lib/ngsiHandlers.js

```diff
@@ -144,7 +144,8 @@ function selectRealName(extractedName, device, callback) {
  * @param {String} type         Type of the entity.
  * @param {Array} attributes    List of attributes of the entity to be updated (including type and value).
  */
-function ngsiUpdateHandler(id, type, attributes, callback) {
+function ngsiUpdateHandler(id, type, service, subservice, attributes, callback) {
```

Comment:
This PR incluces changes in program code, along with tests. Anything worth to mention in CHANGES_NEXT_RELEASE (i.e. some bug being fixed in the process of fixing the tests) or only trivial changes?

Reply:
Fixed in 0de8f251d1c82832cb5ee4b3104fd86100517c3d